.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -1,6 +1,6 @@
 ---
-name: Bug report
-about: Create a bug report to help us improve
+name: Bug Report - Netdata Agent
+about: Submit a report and help us improve our Open-Source free Netdata Agent
```

Comment:
If you really want to include "free," please use "free and open-source," which is the more common (FOSS) usage.

```suggestion
about: Submit a report and help us improve our free and open-source Netdata Agent
```